crates/derive/src/combine_api.rs

```diff
@@ -81,6 +86,12 @@ impl cgo::Service for API {
                     field_ptrs,
                     uuid_placeholder_ptr,
                 } = combine_api::Config::decode(data)?;
+                tracing::debug!(
```

Comment:
nit: spacing here, and other macro calls that weren't formatted.

Reply:
This doesn't add anything not already in the tracing::debug line which immediately follows this one (also, formatting still).